### PR TITLE
Refactor and change project cards

### LINKED ISSUE #352: ui: Make project card scrollable

### LINKED ISSUE #354: ui: Adapt size and format of cards in project overview
Resize cards and change style, so all the data is visible.

### LINKED ISSUE #357: ui: Create new component for project card
Could be done together with #354

### PR DESCRIPTION
**Pull request by mathiashoeld**
# Changes

* Create new component for project card
* Set fixed height of project cards
* Make project cards slightly wider
* Make content of project cards scrollable
* Refactor styles

Closes #354 
Closes #352 
Closes #357

### CODE REVIEW

File: frontend/src/pages/Overview/OverviewTable.js [hunk 1]
```diff
@@ -146,139 +156,43 @@ const getTableEntries = ({
...
-            display: "flex",
-            backgroundColor: "lightgray",
-            flexDirection: "row",
-            maxWidth: "350px",
-            height: "100%",
-            alignItems: "center",
-            justifyContent: "center"
-          }}
-        >
+      <Card data-test="projectcreation" style={styles.addProject}>
```
Comment (Stezido) [inline frontend/src/pages/Overview/OverviewTable.js]:
```suggestion
<Card data-test="project-creation" style={styles.addProject}>
```

all of our data-test tags are written seperated by hyphens

File: frontend/src/pages/Overview/ProjectCard.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  editDisabled,
+  showEditDialog,
+  description,
+  thumbnail,
+  tags,
+  classes,
+  imagePath
+}) => {
+  return (
+    <Card aria-label="project" key={index} className={classes.card} data-test={`projectcard-${id}`}>
```
Comment (Stezido) [inline frontend/src/pages/Overview/ProjectCard.js]:
```suggestion
 <Card aria-label="project" key={index} className={classes.card} data-test={`project-card-${id}`}>
```

File: frontend/src/pages/Overview/ProjectCard.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+              data-test={`project-view-button-${index}`}
+            >
+              <ViewIcon />
+            </Fab>
+          </div>
+        </Tooltip>
+      </CardActions>
+      <CardContent>
+        <CardHeader
+          data-test="projectheader"
```
Comment (Stezido) [inline frontend/src/pages/Overview/ProjectCard.js]:
```suggestion
data-test="project-header"
```

File: frontend/src/pages/Overview/ProjectCard.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+        <List>
+          <div
+            style={{ marginTop: "5px", height: "200px", overflow: "scroll", overflowY: "auto", overflowX: "hidden" }}
+          >
+            {projectedBudgets.length === 0 ? null : (
+              <ListItem className={classes.listItem} disabled={true}>
+                <ListItemIcon>
+                  <AmountIcon />
+                </ListItemIcon>
+                <ListItemText data-test="projectbudget" primary={budgets} secondary={strings.common.budget} />
```
Comment (Stezido) [inline frontend/src/pages/Overview/ProjectCard.js]:
```suggestion
<ListItemText data-test="project-budget" primary={budgets} secondary={strings.common.budget} />
```

File: frontend/src/pages/Overview/ProjectCard.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+              </ListItem>
+            )}
+            <ListItem className={classes.listItem} disabled={true}>
+              <ListItemIcon>
+                <DateIcon />
+              </ListItemIcon>
+              <ListItemText data-test="projectcreation" primary={dateString} secondary={strings.common.created} />
```
Comment (Stezido) [inline frontend/src/pages/Overview/ProjectCard.js]:
```suggestion
<ListItemText data-test="project-creation" primary={dateString} secondary={strings.common.created} />
```

File: frontend/src/pages/Overview/OverviewTable.js (hunk 1 above)
Comment (mathiashoeld) [reply frontend/src/pages/Overview/OverviewTable.js]: changed to "project-creation"

File: frontend/src/pages/Overview/ProjectCard.js (hunk 2 above)
Comment (mathiashoeld) [reply frontend/src/pages/Overview/ProjectCard.js]: Changed to project-card

File: frontend/src/pages/Overview/ProjectCard.js (hunk 3 above)
Comment (mathiashoeld) [reply frontend/src/pages/Overview/ProjectCard.js]: Changed to "project-header"

File: frontend/src/pages/Overview/ProjectCard.js (hunk 4 above)
Comment (mathiashoeld) [reply frontend/src/pages/Overview/ProjectCard.js]: changed to "project-budget"

File: frontend/src/pages/Overview/ProjectCard.js (hunk 5 above)
Comment (mathiashoeld) [reply frontend/src/pages/Overview/ProjectCard.js]: done (see comment above)

**Pull request merged**